### ISSUE TITLE
Attribute Interpolation failed

### ISSUE THREAD

**Issue by yoyoys** [Ty3uK/parcel-plugin-pug#10]
I used variable for define relative path for resource.
and it's okay when compiled with gulp.
but i tried compiled with parcel, i got this error message, i think it's a wrong compile.

I'm using parcel-plugin-pug 0.2.5.

here is my structure:
```
_layout.pug
Test/
--test.pug
```
here is my code:
_layout.pug
```pug
block vars
  -var basePath = '';

<!DOCTYPE html>
html(lang="en")
  head
    title Document
  body
    block content
    script(src=basePath+'assets/js/test1.js')
```
test.pug
```pug
extends ../_layout.pug

block vars
  -var base = '../'
block content
  h1 test
```
and here is the error :
![image](https://user-images.githubusercontent.com/22124891/36573371-8cebb040-187b-11e8-9ecc-b9fa21ad7b1c.png)

It compiled as `'./asePath+%27assets/js/test1.js'`

**Comment by Ty3uK** [comment]
Yep, I know that problem. Fix it ASAP, thanks!

**Comment by dolma** [comment]
Had a similar problem with an `href` tag so I made a PR to fix it (#16)

**Comment by Ty3uK** [comment]
Fixed in PR, thanks @dolma!
